docs/contribute-to-romcal.md

```diff
@@ -125,3 +125,51 @@ _pass the `-t` flag and the path of the file_
 ## Coverage reporting
 
 A coverage report is generated for every core branch build and published to [gh-pages]().
+
+## Committing changes
+
+### Message Format
+
+When you are ready to commit your changes, your commit message should follow the conventional commit format
+
+General format/terminology:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+Here are some guardrails to follow:
+
+When "type" one of the keys, the scopes allowed are as follows:
+
+- `ci` (updates to automation or the release process): No scope allowed
+- `build` (updates to build scripts): No scope allowed
+- `docs` (documentation): No Scope allowed
+- `feat` (net-new feature):
+  - Any of the calendar ids (`general` or `europe`, or `slovakia`, etc.)
+  - Any of the locale ids (`en`, `fr`, `es`, etc.)
+  - 'l10n' for changes to localization, either multiple locales or not specific to one
+  - 'calendar' for changes to calendars, either multiple or not specific to one
+- `enh` (enhancement): see `feat`
+- `fix` (bug fix): see `feat`
+- `refactor` (refactoring of code): see `feat`
+- `perf` (performance improvements): see `feat`
+- `test` (adding or updating tests): see `feat`
+- `style` (formatting, missing semi-colons, etc; no code change): see for `feat`
```

Comment:
I usually include changes related to formatting, linting and grammar.

Reply:
I’d rather write more examples, however, feel free to keep it as it. :wink:

Reply:
i can add some more while i clarify the description of the whole list

Reply:
That said, maybe I should start to be less nit-picky, as there are much more important things to do with romcal.